packages/nuka/src/carousel.tsx

```diff
@@ -646,7 +646,7 @@ export const Carousel = (rawProps: CarouselProps): React.ReactElement => {
         }}
         aria-label={frameAriaLabel}
         role="region"
-        tabIndex={0}
+        tabIndex={enableKeyboardControls ? 0 : undefined}
```

Comment:
```suggestion
        tabIndex={enableKeyboardControls ? 0 : -1}
```

Let's fall back to -1, just so we don't break any existing logic that programmatically focuses on this component.